client/main.lua

```diff
@@ -243,8 +243,8 @@ function GiveKeys(id, plate)
 end
 
 function GetKeys()
-    lib.callback('qb-vehiclekeys:server:GetVehicleKeys', function(keysList)
-        KeysList = keysList
+    QBCore.Functions.TriggerCallback('qb-vehiclekeys:server:GetVehicleKeys', function(keysList)
```

Comment:
Not sure why you changed this back to qb callbacks. They are deprecated. Please change back to lib callback